src/components/icon-list-row/row-info.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+
+const RowInfo = ({ children, bottom }) => {
+  const styles = {
+    display: 'flex',
+    justifyContent: 'space-between',
+    paddingTop: '0.25rem',
+    paddingBottom: '0.25rem',
+    alignItems: (bottom ? null : 'baseline'),
+  };
+
+  return (
+    <div style={styles}>
+      {children[0]}
```

Comment:
This seems a bit odd imo. Either we should force it to _exactly_ two "child components" by not using the `children` prop or we should accept any number of children.